reggie/macros.jinja

```diff
@@ -159,3 +159,11 @@ include:
 
 {{ encode_ini(data) }}
 {%- endmacro -%}
+
+{%- macro handle_windows_permissions() -%}
+    {% if grains['is_vagrant_windows'] | default(false) %}
+    # on windows, in vboxfs shared filesystems (used to share from win->linux), this is unchangable.
+    # set it here so that salt won't fail when it tries to save. kind of a hack, but only on vagrant so... whatever
+    - mode: 0777
+    {% endif %}
+{%- endmacro -%}
```

Comment:
I know 0777 seems bad, but, it's vagrant+windows and that vboxfs won't allow anything EXCEPT 0777, so it's fine to set it here.   don't do this on production ever.

Reply:
That seems fine - my only feedback here is to use ``{# ... #}`` jinja comments instead of ``#`` comments, since the latter will get injected into the state at a weird indentation level which is unlikely to cause a problem in practice but theoretically could.

I'll make that change before testing on my local branch.